### ISSUE TITLE
A11y_VS Code: Copilot_Suggestion_Keyboard: Tooltip is not accessible using keyboard.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#27361]
-A11yTCS; -A11ySev2; -DesktopApp; -Win11; -Win32; -A11yWCAG2.1; -SH-VS Code Copilot-Win32-July23; -VS Code; -A11yAuto; -Keyboard; -A11yMAS;

--- Environment Details:
Application Name: VS Code: Copilot
Visual studio code Insiders Version: 1.81.0(User Setup)
Microsoft Windows 11 Enterprise 22H2 Build 22621.1992

--- Repro Steps:

1. Open the "Visual Studio Code" dialog box.
2. In Visual Studio Code, create a new JavaScript (*.js) file.
3. In the code Paste this code function calculateDaysBetweenDates(begin, end) { and press enter.
4. A suggestion will appear.
5. Prees "Ctrl + K, Ctrl + I" and a tooltip will appear.
6. Verify whether the tooltip is accessible using keyboard or not.

--- Actual Result:​
Tooltip is not accessible using keyboard.

--- Expected Result:​
Tooltip should be accessible using keyboard.

--- User Impact:
Keyboard users will face difficulty in accessing the tooltip info.

--- Attachment:
